docs/src/pages/components/cards/ActionAreaCard.js

```diff
@@ -10,9 +10,10 @@ export default function ActionAreaCard() {
     <Card sx={{ maxWidth: 345 }}>
       <CardActionArea>
         <CardMedia
-          sx={{ height: 140 }}
+          component="img"
+          height="140"
           image="/static/images/cards/contemplative-reptile.jpg"
-          title="Contemplative Reptile"
+          alt="green iguana"
```

Comment:
Found http://birdandexotic.com/becs-veterinary-care-animals/green-iguana/ via image search. https://www.google.de/search?q=green+iguana&hl=en&sxsrf=ALeKk00E6tkMrtFfI81f4o4gasMRqn3sJA:1628581403130&source=lnms&tbm=isch&sa=X&ved=2ahUKEwigvZXA-qXyAhUH6qQKHZPRA0sQ_AUoAXoECAEQAw&biw=2560&bih=1248 seems to fit.